### PR TITLE
Astro maybe

### PR DESCRIPTION
**Pull request by hougasian**

### CODE REVIEW

File: src/layouts/partials/site/scripts.astro [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
 ---
-<script src="https://unpkg.com/@ritterim/platform-ui/dist/js/platform-ui.min.js" crossorigin defer></script>
+<script src="https://cdn.jsdelivr.net/npm/@@ritterim/platform-ui/dist/platform-ui.min.js" crossorigin defer></script>
```
Comment (mrtrimble) [inline src/layouts/partials/site/scripts.astro:3]: need this double `@@`?

File: src/layouts/partials/site/scripts.astro [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
 <script type="text/javascript" src="https://unpkg.com/@ritterim/skellycss/dist/skelly.js" crossorigin defer></script>
```
[the remark this answers is not in the document]
Comment (tedk13) [reply src/layouts/partials/site/scripts.astro:4]: Yes

[the remark this answers is not in the document]
Comment (tedk13) [reply src/layouts/partials/site/scripts.astro:4]: No more unpkg!

File: src/layouts/partials/site/scripts.astro (hunk 1 above)
Comment (mrtrimble) [reply src/layouts/partials/site/scripts.astro:3]: URL Is:
```suggestion
<script src="https://cdn.jsdelivr.net/npm/@ritterim/platform-ui/dist/js/platform-ui.min.js" crossorigin defer></script>
```

**Review by mrtrimble — APPROVED** [review]

**Pull request merged**